01-import_and_maxfilter.py

```diff
@@ -97,6 +97,48 @@ def rename_events(raw, subject, session):
         description[description == old_event_name] = new_event_name
 
 
+def plot_auto_scores(auto_scores):
+    import matplotlib.pyplot as plt
+    import seaborn as sns
+
+    if config.ch_types == ['meg']:
+        ch_types = ['grad', 'mag']
+    else:
+        ch_types = config.ch_types
+
+    for ch_type in ch_types:
+        # Only select the data for mag or grad channels.
+        ch_subset = auto_scores['ch_types'] == ch_type
+        ch_names = auto_scores['ch_names'][ch_subset]
+        scores = auto_scores['scores_noisy'][ch_subset]
+        limits = auto_scores['limits_noisy'][ch_subset]
+        # Number of time windows
+        segments = range(1, len(auto_scores['bin_edges']))
+
+        data_to_plot = pd.DataFrame(data=scores,
+                                    columns=pd.Index(segments, name='Segment'),
+                                    index=pd.Index(ch_names, name='Channel'))
+
+        # Plot the "raw" scores.
+        fig, ax = plt.subplots(1, 2, figsize=(12, 5))
+        fig.suptitle(f'Automated noisy channel detection: {ch_type}',
+                     fontsize=16, fontweight='bold')
+        sns.heatmap(data=data_to_plot, cmap='Reds',
+                    cbar_kws=dict(label='Score'),
+                    ax=ax[0])
+        ax[0].set_title('All Scores', fontweight='bold')
+
+        # Focus on scores that exceeded the limits.
+        sns.heatmap(data=data_to_plot,
+                    vmin=limits.min(), cmap='Reds',
+                    cbar_kws=dict(label='Score'), ax=ax[1])
+        ax[1].set_title('Scores > Limit', fontweight='bold')
+
+        # Figure title should not overlap with subplots.
+        fig.tight_layout(rect=[0, 0.03, 1, 0.95])
+        plt.show()
```

Comment:
why is this not in MNE?

Reply:
Hey @jasmainak, so the code in this PR is based on what can be found in the Maxwell filter tutorial of MNE, see https://mne.tools/dev/auto_tutorials/preprocessing/plot_60_maxwell_filtering_sss.html#using-sss-and-maxwell-filtering-in-mne-python

You can also find an example plot there.

The reason this is currently not in "MNE proper" is that we haven't decided how to best implement this. So I'd consider the current code in this PR as temporary, until we have a better solution built right into MNE!